diff -x fix.patch -Naur FileUnknown.orig/FileUnknown-0.ebuild FileUnknown/FileUnknown-0.ebuild
--- FileUnknown.orig/FileUnknown-0.ebuild	2025-04-15 17:10:26.943486895 -0600
+++ FileUnknown/FileUnknown-0.ebuild	2025-04-15 17:10:22.086456665 -0600
@@ -8,11 +8,10 @@
 
 PATCHES=(
 	"${FILESDIR}"/existent.patch
-	"${FILESDIR}"/nonexistent.patch
 )
 
 # invalid references in strings are flagged
-FILE1="file1:${FILESDIR}/nonexistent"
+FILE1="file1:${FILESDIR}/existent"
 FILE2="file2:${FILESDIR}/existent"
 
 src_prepare() {
