diff -Naur LicenseUnneeded.orig/LicenseUnneeded-1.ebuild LicenseUnneeded/LicenseUnneeded-1.ebuild
--- LicenseUnneeded.orig/LicenseUnneeded-1.ebuild	2024-06-07 01:17:51.112658803 -0600
+++ LicenseUnneeded/LicenseUnneeded-1.ebuild	2024-06-07 01:22:32.948604058 -0600
@@ -1,6 +1,5 @@
 EAPI=8
 DESCRIPTION="Virtual ebuild without a license"
 HOMEPAGE="https://pkgcraft.pkgcraft"
-LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64"
