diff -Naur DependencyInvalid.orig/DependencyInvalid-0.ebuild DependencyInvalid/DependencyInvalid-0.ebuild
--- DependencyInvalid.orig/DependencyInvalid-0.ebuild	2024-07-06 01:07:12.161771000 -0600
+++ DependencyInvalid/DependencyInvalid-0.ebuild	2024-07-06 11:24:51.186606628 -0600
@@ -7,5 +7,5 @@
 KEYWORDS="amd64"
 DEPEND="
 	stub/slotted:0=
-	stub/slotted:1/0=
+	stub/slotted:1=
 "
diff -Naur DependencyInvalid.orig/DependencyInvalid-1.ebuild DependencyInvalid/DependencyInvalid-1.ebuild
--- DependencyInvalid.orig/DependencyInvalid-1.ebuild	2024-07-06 11:22:01.526520978 -0600
+++ DependencyInvalid/DependencyInvalid-1.ebuild	2024-07-06 11:25:02.360612269 -0600
@@ -5,4 +5,4 @@
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64"
-PDEPEND="stub/slotted:0="
+PDEPEND="stub/slotted:0"
