diff -Naur DependencyInvalid.orig/DependencyInvalid-0.ebuild DependencyInvalid/DependencyInvalid-0.ebuild
--- DependencyInvalid.orig/DependencyInvalid-0.ebuild	2024-07-06 12:42:42.221942944 -0600
+++ DependencyInvalid/DependencyInvalid-0.ebuild	2024-07-06 12:47:17.244093735 -0600
@@ -7,7 +7,7 @@
 KEYWORDS="amd64"
 IUSE="u1 u2 u3"
 DEPEND="
-	u1? ( stub/slotted:1/0= )
+	u1? ( stub/slotted:1= )
 	u2? ( stub/slotted:0= )
-	u3? ( stub/slotted:1/0= )
+	u3? ( stub/slotted:1= )
 "
diff -Naur DependencyInvalid.orig/DependencyInvalid-1.ebuild DependencyInvalid/DependencyInvalid-1.ebuild
--- DependencyInvalid.orig/DependencyInvalid-1.ebuild	2024-07-06 11:22:01.526520978 -0600
+++ DependencyInvalid/DependencyInvalid-1.ebuild	2024-07-06 12:47:30.334237881 -0600
@@ -5,4 +5,4 @@
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64"
-PDEPEND="stub/slotted:0="
+PDEPEND="stub/slotted:0"
