diff -x fix.patch -Naur CommandScopeInvalid.orig/CommandScopeInvalid-0.ebuild CommandScopeInvalid/CommandScopeInvalid-0.ebuild
--- CommandScopeInvalid.orig/CommandScopeInvalid-0.ebuild	2025-02-26 17:07:13.831067972 -0700
+++ CommandScopeInvalid/CommandScopeInvalid-0.ebuild	2025-02-26 17:06:57.696986444 -0700
@@ -7,5 +7,8 @@
 
 src_compile() {
 	emake program
+}
+
+src_install() {
 	dobin program
 }
