diff -x fix.patch -Naur HomepageInvalid.orig/HomepageInvalid-0.ebuild HomepageInvalid/HomepageInvalid-0.ebuild
--- HomepageInvalid.orig/HomepageInvalid-0.ebuild	2024-12-22 12:29:45.423419927 -0700
+++ HomepageInvalid/HomepageInvalid-0.ebuild	2024-12-22 12:29:33.260346807 -0700
@@ -1,5 +1,6 @@
 EAPI=8
 
 DESCRIPTION="ebuild with missing HOMEPAGE"
+HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
 SLOT="0"
diff -x fix.patch -Naur HomepageInvalid.orig/HomepageInvalid-1.ebuild HomepageInvalid/HomepageInvalid-1.ebuild
--- HomepageInvalid.orig/HomepageInvalid-1.ebuild	2024-12-22 12:29:45.423419927 -0700
+++ HomepageInvalid/HomepageInvalid-1.ebuild	2024-12-22 12:29:36.586366801 -0700
@@ -1,6 +1,6 @@
 EAPI=8
 
 DESCRIPTION="ebuild with HOMEPAGE using unsupported protocol"
-HOMEPAGE="ftp://pkgcraft.pkgcraft"
+HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
 SLOT="0"
diff -x fix.patch -Naur HomepageInvalid.orig/HomepageInvalid-2.ebuild HomepageInvalid/HomepageInvalid-2.ebuild
--- HomepageInvalid.orig/HomepageInvalid-2.ebuild	2024-12-22 12:29:45.423419927 -0700
+++ HomepageInvalid/HomepageInvalid-2.ebuild	2024-12-22 12:29:42.288401082 -0700
@@ -1,6 +1,6 @@
 EAPI=8
 
 DESCRIPTION="ebuild with invalid HOMEPAGE URL"
-HOMEPAGE="pkgcraft"
+HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
 SLOT="0"
