diff -x fix.patch -Naur MetadataError.orig/MetadataError-0.ebuild MetadataError/MetadataError-0.ebuild
--- MetadataError.orig/MetadataError-0.ebuild	2025-09-20 17:58:38.305917054 -0600
+++ MetadataError/MetadataError-0.ebuild	2025-09-20 17:55:58.207800427 -0600
@@ -1,6 +1,5 @@
 EAPI=8
 
-cd dir
 DESCRIPTION="restricted builtin"
 HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
diff -x fix.patch -Naur MetadataError.orig/MetadataError-0-r1.ebuild MetadataError/MetadataError-0-r1.ebuild
--- MetadataError.orig/MetadataError-0-r1.ebuild	2025-09-20 17:58:38.304917053 -0600
+++ MetadataError/MetadataError-0-r1.ebuild	2025-09-20 17:55:58.207800427 -0600
@@ -1,6 +1,5 @@
 EAPI=8
 
-history
 DESCRIPTION="unsupported builtin (compiled out)"
 HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
diff -x fix.patch -Naur MetadataError.orig/MetadataError-0-r2.ebuild MetadataError/MetadataError-0-r2.ebuild
--- MetadataError.orig/MetadataError-0-r2.ebuild	2025-09-20 17:58:38.304917053 -0600
+++ MetadataError/MetadataError-0-r2.ebuild	2025-09-20 17:55:58.207800427 -0600
@@ -1,6 +1,5 @@
 EAPI=8
 
-enable
 DESCRIPTION="unsupported builtin (explicitly disabled)"
 HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
diff -x fix.patch -Naur MetadataError.orig/MetadataError-0-r3.ebuild MetadataError/MetadataError-0-r3.ebuild
--- MetadataError.orig/MetadataError-0-r3.ebuild	2025-09-20 17:58:38.304917053 -0600
+++ MetadataError/MetadataError-0-r3.ebuild	2025-09-20 17:55:58.207800427 -0600
@@ -1,7 +1,5 @@
 EAPI=8
 
-# recursively source ebuild
-source ${BASH_SOURCE}
 DESCRIPTION="unsupported builtin (explicitly disabled)"
 HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
diff -x fix.patch -Naur MetadataError.orig/MetadataError-0-r4.ebuild MetadataError/MetadataError-0-r4.ebuild
--- MetadataError.orig/MetadataError-0-r4.ebuild	2025-09-20 17:58:38.305917054 -0600
+++ MetadataError/MetadataError-0-r4.ebuild	2025-09-20 17:55:58.207800427 -0600
@@ -1,7 +1,5 @@
 EAPI=8
 
-# source another ebuild
-. MetadataError-0.ebuild
 DESCRIPTION="unsupported builtin (explicitly disabled)"
 HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
diff -x fix.patch -Naur MetadataError.orig/MetadataError-1.ebuild MetadataError/MetadataError-1.ebuild
--- MetadataError.orig/MetadataError-1.ebuild	2025-09-20 17:58:38.305917054 -0600
+++ MetadataError/MetadataError-1.ebuild	2025-09-20 17:55:58.208800428 -0600
@@ -1,6 +1,5 @@
 EAPI=8
 
-https://pkgcraft.pkgcraft
 DESCRIPTION="invalid data"
 HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
diff -x fix.patch -Naur MetadataError.orig/MetadataError-2.ebuild MetadataError/MetadataError-2.ebuild
--- MetadataError.orig/MetadataError-2.ebuild	2025-09-20 17:58:38.305917054 -0600
+++ MetadataError/MetadataError-2.ebuild	2025-09-20 17:55:58.208800428 -0600
@@ -1,6 +1,5 @@
 EAPI=8
 
-/path/to/command
 DESCRIPTION="absolute command path"
 HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
diff -x fix.patch -Naur MetadataError.orig/MetadataError-3.ebuild MetadataError/MetadataError-3.ebuild
--- MetadataError.orig/MetadataError-3.ebuild	2025-09-20 17:58:38.305917054 -0600
+++ MetadataError/MetadataError-3.ebuild	2025-09-20 17:55:58.211800430 -0600
@@ -1,6 +1,6 @@
 EAPI=8
 
-VAR="string missing end quote
+VAR="string missing end quote"
 DESCRIPTION="invalid syntax"
 HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
diff -x fix.patch -Naur MetadataError.orig/MetadataError-4.ebuild MetadataError/MetadataError-4.ebuild
--- MetadataError.orig/MetadataError-4.ebuild	2025-09-20 17:58:38.305917054 -0600
+++ MetadataError/MetadataError-4.ebuild	2025-09-20 17:55:58.211800430 -0600
@@ -1,7 +1,6 @@
 EAPI=8
 
 echo > /dev/null
-echo > test
 DESCRIPTION="redirection disallowed (except to /dev/null)"
 HOMEPAGE="https://pkgcraft.pkgcraft"
 LICENSE="MIT"
diff -x fix.patch -Naur MetadataError.orig/MetadataError-5.ebuild MetadataError/MetadataError-5.ebuild
--- MetadataError.orig/MetadataError-5.ebuild	2025-09-20 17:58:38.305917054 -0600
+++ MetadataError/MetadataError-5.ebuild	2025-09-20 17:58:31.145911703 -0600
@@ -5,4 +5,4 @@
 LICENSE="MIT"
 SLOT="0"
 
-PATCHES=( nonexistent-glob-fail* )
+PATCHES=( nonexistent-glob-fail )
