diff -Naur EbuildNameInvalid.orig/EbuildNameInvalid-0-1.ebuild EbuildNameInvalid/EbuildNameInvalid-0-1.ebuild
--- EbuildNameInvalid.orig/EbuildNameInvalid-0-1.ebuild	2024-12-01 06:40:51.372218533 -0700
+++ EbuildNameInvalid/EbuildNameInvalid-0-1.ebuild	1969-12-31 17:00:00.000000000 -0700
@@ -1,6 +0,0 @@
-EAPI=8
-
-DESCRIPTION="Ebuild with invalid version"
-HOMEPAGE="https://pkgcraft.pkgcraft"
-LICENSE="MIT"
-SLOT="0"
diff -Naur EbuildNameInvalid.orig/EbuildNameInvalid-0.ebuild EbuildNameInvalid/EbuildNameInvalid-0.ebuild
--- EbuildNameInvalid.orig/EbuildNameInvalid-0.ebuild	1969-12-31 17:00:00.000000000 -0700
+++ EbuildNameInvalid/EbuildNameInvalid-0.ebuild	2024-12-01 06:40:51.372218533 -0700
@@ -0,0 +1,6 @@
+EAPI=8
+
+DESCRIPTION="Ebuild with invalid version"
+HOMEPAGE="https://pkgcraft.pkgcraft"
+LICENSE="MIT"
+SLOT="0"
diff -Naur EbuildNameInvalid.orig/EbuildNameInvalid1-2.ebuild EbuildNameInvalid/EbuildNameInvalid1-2.ebuild
--- EbuildNameInvalid.orig/EbuildNameInvalid1-2.ebuild	2024-12-01 06:42:12.505430056 -0700
+++ EbuildNameInvalid/EbuildNameInvalid1-2.ebuild	1969-12-31 17:00:00.000000000 -0700
@@ -1,6 +0,0 @@
-EAPI=8
-
-DESCRIPTION="Ebuild with mismatched name"
-HOMEPAGE="https://pkgcraft.pkgcraft"
-LICENSE="MIT"
-SLOT="0"
diff -Naur EbuildNameInvalid.orig/EbuildNameInvalid-1-a.ebuild EbuildNameInvalid/EbuildNameInvalid-1-a.ebuild
--- EbuildNameInvalid.orig/EbuildNameInvalid-1-a.ebuild	2024-12-01 06:41:21.058290405 -0700
+++ EbuildNameInvalid/EbuildNameInvalid-1-a.ebuild	1969-12-31 17:00:00.000000000 -0700
@@ -1,6 +0,0 @@
-EAPI=8
-
-DESCRIPTION="Ebuild with invalid version"
-HOMEPAGE="https://pkgcraft.pkgcraft"
-LICENSE="MIT"
-SLOT="0"
diff -Naur EbuildNameInvalid.orig/EbuildNameInvalid-1.ebuild EbuildNameInvalid/EbuildNameInvalid-1.ebuild
--- EbuildNameInvalid.orig/EbuildNameInvalid-1.ebuild	1969-12-31 17:00:00.000000000 -0700
+++ EbuildNameInvalid/EbuildNameInvalid-1.ebuild	2024-12-01 06:41:21.058290405 -0700
@@ -0,0 +1,6 @@
+EAPI=8
+
+DESCRIPTION="Ebuild with invalid version"
+HOMEPAGE="https://pkgcraft.pkgcraft"
+LICENSE="MIT"
+SLOT="0"
diff -Naur EbuildNameInvalid.orig/EbuildNameInvalid-2.ebuild EbuildNameInvalid/EbuildNameInvalid-2.ebuild
--- EbuildNameInvalid.orig/EbuildNameInvalid-2.ebuild	1969-12-31 17:00:00.000000000 -0700
+++ EbuildNameInvalid/EbuildNameInvalid-2.ebuild	2024-12-01 06:42:12.505430056 -0700
@@ -0,0 +1,6 @@
+EAPI=8
+
+DESCRIPTION="Ebuild with mismatched name"
+HOMEPAGE="https://pkgcraft.pkgcraft"
+LICENSE="MIT"
+SLOT="0"
diff -Naur EbuildNameInvalid.orig/EbuildNameInvalid-3.ebuild EbuildNameInvalid/EbuildNameInvalid-3.ebuild
--- EbuildNameInvalid.orig/EbuildNameInvalid-3.ebuild	1969-12-31 17:00:00.000000000 -0700
+++ EbuildNameInvalid/EbuildNameInvalid-3.ebuild	2024-12-01 06:53:22.623372998 -0700
@@ -0,0 +1,6 @@
+EAPI=8
+
+DESCRIPTION="Ebuild with missing version"
+HOMEPAGE="https://pkgcraft.pkgcraft"
+LICENSE="MIT"
+SLOT="0"
diff -Naur EbuildNameInvalid.orig/EbuildNameInvalid.ebuild EbuildNameInvalid/EbuildNameInvalid.ebuild
--- EbuildNameInvalid.orig/EbuildNameInvalid.ebuild	2024-12-01 06:53:22.623372998 -0700
+++ EbuildNameInvalid/EbuildNameInvalid.ebuild	1969-12-31 17:00:00.000000000 -0700
@@ -1,6 +0,0 @@
-EAPI=8
-
-DESCRIPTION="Ebuild with missing version"
-HOMEPAGE="https://pkgcraft.pkgcraft"
-LICENSE="MIT"
-SLOT="0"
