diff -x fix.patch -Naur EbuildVersionsEqual.orig/EbuildVersionsEqual-000.ebuild EbuildVersionsEqual/EbuildVersionsEqual-000.ebuild
--- EbuildVersionsEqual.orig/EbuildVersionsEqual-000.ebuild	2024-12-10 22:38:00.068568474 -0700
+++ EbuildVersionsEqual/EbuildVersionsEqual-000.ebuild	1969-12-31 17:00:00.000000000 -0700
@@ -1,6 +0,0 @@
-EAPI=8
-
-DESCRIPTION="Ebuild with equivalent version"
-HOMEPAGE="https://pkgcraft.pkgcraft"
-LICENSE="MIT"
-SLOT="0"
diff -x fix.patch -Naur EbuildVersionsEqual.orig/EbuildVersionsEqual-0-r00.ebuild EbuildVersionsEqual/EbuildVersionsEqual-0-r00.ebuild
--- EbuildVersionsEqual.orig/EbuildVersionsEqual-0-r00.ebuild	2024-12-10 22:38:00.068568474 -0700
+++ EbuildVersionsEqual/EbuildVersionsEqual-0-r00.ebuild	1969-12-31 17:00:00.000000000 -0700
@@ -1,6 +0,0 @@
-EAPI=8
-
-DESCRIPTION="Ebuild with equivalent version"
-HOMEPAGE="https://pkgcraft.pkgcraft"
-LICENSE="MIT"
-SLOT="0"
diff -x fix.patch -Naur EbuildVersionsEqual.orig/EbuildVersionsEqual-0-r0.ebuild EbuildVersionsEqual/EbuildVersionsEqual-0-r0.ebuild
--- EbuildVersionsEqual.orig/EbuildVersionsEqual-0-r0.ebuild	2024-12-10 22:38:00.068568474 -0700
+++ EbuildVersionsEqual/EbuildVersionsEqual-0-r0.ebuild	1969-12-31 17:00:00.000000000 -0700
@@ -1,6 +0,0 @@
-EAPI=8
-
-DESCRIPTION="Ebuild with equivalent version"
-HOMEPAGE="https://pkgcraft.pkgcraft"
-LICENSE="MIT"
-SLOT="0"
