diff -x fix.patch -Naur qa-primary.orig/CategoryEmpty/PackageEmpty/.empty qa-primary/CategoryEmpty/PackageEmpty/.empty
--- qa-primary.orig/CategoryEmpty/PackageEmpty/.empty	2024-12-19 05:01:40.579386426 -0700
+++ qa-primary/CategoryEmpty/PackageEmpty/.empty	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-# purposely empty package dir
diff -x fix.patch -Naur qa-primary.orig/licenses/deprecated qa-primary/licenses/deprecated
--- qa-primary.orig/licenses/deprecated	2024-12-19 05:01:40.579386426 -0700
+++ qa-primary/licenses/deprecated	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-# deprecated stub license
diff -x fix.patch -Naur qa-primary.orig/profiles/categories qa-primary/profiles/categories
--- qa-primary.orig/profiles/categories	2024-12-19 05:01:40.579386426 -0700
+++ qa-primary/profiles/categories	2024-12-19 05:01:30.725312655 -0700
@@ -1,5 +1,4 @@
 Builtins
-CategoryEmpty
 Dependency
 DependencySlotMissing
 EapiStale
